provdebug/ProvSlice.py

```diff
@@ -25,7 +25,8 @@ def __init__(self, nodes: List[ProvNode]):
     def compute_similarity(self, other_slice: 'ProvSlice') -> float:
         self_content = ''.join([node.content for node in self._nodes]).replace(" ", "")
         other_content = ''.join([node.content for node in other_slice._nodes]).replace(" ", "")
-        return SequenceMatcher(None, self_content, other_content).ratio()
+        similarity_score = SequenceMatcher(None, self_content, other_content).ratio()
+        return round(similarity_score, 2)
```

Comment:
We display the similarity score to the user now so that they can decide what to set the new similarity threshold to in order to make sure they get a result. Round the score to 2 decimal places for readability?

Reply:
Sounds like a good idea!